common/etc/profile

```diff
@@ -0,0 +1,36 @@
+# /etc/profile: system-wide .profile file for the Bourne shell (sh(1))
+# and Bourne compatible shells (bash(1), ksh(1), ash(1), ...).
+
+if [ "`id -u`" -eq 0 ]; then
+    PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+else
+    PATH="/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games"
+fi
+export PATH
+
+if [ "$PS1" ]; then
+    if [ "$BASH" ] && [ "$BASH" != "/bin/sh" ]; then
+	# The file bash.bashrc already sets the default PS1.
+        # PS1='\h:\w\$ '
+	if [ -f /etc/bash.bashrc ]; then
+	    . /etc/bash.bashrc
+	fi
+    else
+	if [ "`id -u`" -eq 0 ]; then
+	    PS1='# '
+	else
+	    PS1='$ '
+	fi
+    fi
+fi
+
+if [ -d /etc/profile.d ]; then
+    for i in /etc/profile.d/*.sh; do
+	if [ -r $i ]; then
+            . $i
+	fi
+    done
+    unset i
+fi
+
+. /etc/opx/opx-environment.sh
```

Comment:
It seems we should add a file to /etc/profile.d instead of rewriting /etc/profile.